### DISCUSSION TITLE
固定分辨率以固定布局

### DISCUSSION THREAD

**Discussion by RicardoDlee** [shion-app/shion#57] [Q&A]
计算机应用毫无基础，不明白原理，下意识猜测可能是分辨率固定的因素？场景主要是经常会用外接显示器办公，但是显示器和笔记本主机的分辨率不一致，且每天会有几次断开显示器带着笔电去开会的情况，每次断开重连都会导致shion的界面大小、文字大小以及首页展示组件布局的变化，能否通过自适应或其他方式解决？

**Comment by hanaTsuk1** [comment]
布局变了就是窗口大小变了，我没用过外接显示屏，你看能不能在设置里调一下屏幕相关的参数`分辨率`、`缩放`之类的
或者窗口大小都保持在四列的宽度，应该就没问题🤔

**Reply by hanaTsuk1** [reply]
https://github.com/tauri-apps/tauri/issues/6843
